ModeratorFrontEnd/AIForOrcas/AIForOrcas.DTO/Pagination/CandidateFilterOptionsDTO.cs

```diff
@@ -1,12 +1,15 @@
-﻿namespace AIForOrcas.DTO
+﻿using System;
+
+namespace AIForOrcas.DTO
 {
 	public class CandidateFilterOptionsDTO : IFilterOptions
 	{
 		public string SortOrder { get; set; }
 		public string SortBy { get; set; }
 		public string Timeframe { get; set; }
 		public string Location { get; set; }
-
-		public string QueryString { get => $"sortBy={SortBy}&sortOrder={SortOrder}&timeframe={Timeframe}&location={Location}"; }
+		public DateTime? DateFrom { get; set; }
+		public DateTime? DateTo { get; set; }
+		public string QueryString { get => $"sortBy={SortBy}&sortOrder={SortOrder}&timeframe={Timeframe}&location={Location}&DateFrom={DateFrom}&DateTo={DateTo}"; }
```

Comment:
How would this affect the actual query if not DateFrom or DateTo are provided (i.e. range not selected)? The other parameters are required and I don't know if the API would be unhappy receiving empty parameters.

Reply:
When "Select Date Range" is selected and no data is given for "From" and "To" Date the URL will look like this "api/detections/unreviewed?page=1&recordsPerPage=5&sortBy=timestamp&sortOrder=desc&timeframe=**range**&location=all&DateFrom=&DateTo=",

When "All" is selected Date the URL will look like this 
"api/detections/unreviewed?page=1&recordsPerPage=5&sortBy=timestamp&sortOrder=desc&timeframe=**all**&location=all&DateFrom=&DateTo="

API is working fine with these inputs/url. Also when no data is provided for "**From**" and "**To**" date with **"Range selection"** ,API in this case will return all data ( Same as Timeframe selected as "ALL"), Let me know if this behavior does not sound good, i can make the desired changes.

Also, i have added a validation at API where, when both "From" and "To" date are not null then From date should fall before To date.